src/components/SecondaryPanes/Breakpoints.css

```diff
@@ -77,7 +77,10 @@ html .breakpoints-list .breakpoint.paused {
 }
 
 .breakpoint-snippet {
+  overflow: hidden;
+  text-overflow: ellipsis;
   padding-inline-start: 18px;
+  padding-right: 18px;
```

Comment:
I think this should be `padding-inline-end` unless bi-direction does not work there...